README.md

```diff
@@ -181,17 +185,53 @@ The Cloudant and Twitter settings will be populated when you execute `npm run se
   "TWITTER_CONSUMER_SECRET": <populated by `npm run setup`>,
   "TWITTER_ACCESS_TOKEN": <populated by `npm run setup`>,
   "TWITTER_ACCESS_SECRET": <populated by `npm run setup`>,
-  "TWITTER_LISTEN_FOR": "populated by `npm run setup`",
-  "TWITTER_FILTER_FROM": "populated by `npm run setup`",
-  "TWITTER_FILTER_CONTAINING": "populated by `npm run setup`",
+  "TWITTER_LISTEN_FOR": "<populated by `npm run setup`>",
+  "TWITTER_FILTER_FROM": "<populated by `npm run setup`>",
+  "TWITTER_FILTER_CONTAINING": "<populated by `npm run setup`>",
   "TWITTER_PROCESS_RETWEETS": false,
   "TWITTER_RECEIVER_START_AT_BOOT": true,
-  "TWITTER_CHATBOT_SCREENNAME": <populated by `npm run setup`,
+  "TWITTER_CHATBOT_SCREENNAME": <populated by `npm run setup`>,
   "TWITTER_CHATBOT_START_AT_BOOT": true
 }
 
 ```
 
+* If the service credentials from IBM Watson Assistant is IAM based as below, populate the IAM apikey, url, and workspace_id in below JSON and replace the existing content of `env-vars.json` with this one.
+![](https://github.com/IBM/pattern-images/raw/master/watson-assistant/watson_assistant_api_key.png)
+
+```
+{
+  "CLOUDANT_CONNECTION_URL": <populated by `npm run setup`>,
+  "CLOUDANT_USERNAME": <populated by `npm run setup`>,
+  "CLOUDANT_PASSWORD": <populated by `npm run setup`>,
+  "CLOUDANT_ANALYSIS_DB_NAME": "analysis-db",
+  "CLOUDANT_CONVERSATION_STATE_DB_NAME": "conversation-state-db",
+
+  "CONVERSATION_API_URL": "<Use URL here>",
+  "CONVERSATION_API_USER": "apikey",
```

Comment:
I guess this is a little confusing, since api_key goes in both API_USER and API_PASSWORD.
Maybe we should just add a CONVERSATION_API_KEY and change conversation.js to 
```
iam_apikey: process.env.CONVERSATION_API_KEY
```
?
It seems in the other patterns, we chose one way or the other, and asked the user to delete what wasn't being used?

Reply:
apikey is the string used as USER, its used by loopback for basic authentication

Reply:
similar to the one in `serverless-home-automation`